packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -569,6 +596,12 @@ export const start = async (zcf, privateArgs, baggage) => {
       getSchedules() {
         return E(scheduler).getSchedule();
       },
+      getScheduleUpdates() {
+        return scheduleSubscriber;
+      },
+      getBookDataUpdates(brand) {
+        return subscribers.get(brand);
+      },
```

Comment:
please use the `getPublicTopics` style which will aid documentation https://github.com/Agoric/agoric-sdk/blob/013fff8bdc7bb3e8fbecd6c06b59e065ed706e50/packages/inter-protocol/src/price/fluxAggregator.js#L312-L333

Reply:
Hmm. They shouldn't be properties of this object because it's an open set. My first thought is to have an object for each book with `getPublicTopics` as a call on it, but it looks like "book" is a concept internal to the auctioneer. You could have `getPublicTopics` take an optional brand which would give you the topics for that brand book instead of the whole auctioneer.

It's worth considering though having a public facet on the AuctionBook exo and having an Auctioneer method to get one of those by brand.

Either way, I think the book's subscriber is a concern of the book and should be within its own Exo object. I.e. delete the `subscribers` map and get the entry from the book exo. That will also simplify upgrade.

Reply:
Done. Thanks for the suggestion. That seems cleaner.